### PR TITLE
Ports some borg sprites from citadel

### PR DESCRIPTION
**Pull request by thelaughingbomb**
## About The Pull Request

Give you a choice of sprites when picking modules as a borg.

## Why It's Good For The Game

Variety is good.

## Changelog
:cl:
add: Borg variety
/:cl:

### CODE REVIEW

**Comment by coolsurf6** [comment]
Pretty cool idea. What are some of the sprites added though? Could you post them Or where could I find the sprite files to have a look at?

File: code/modules/mob/living/silicon/robot/robot_modules.dm [hunk 1]
```diff
@@ -547,6 +640,10 @@
 			cyborg_base_icon = "tophat"
 			special_light_key = null
 			hat_offset = INFINITY //He is already wearing a hat
+		if("Heavy")
+			cyborg_base_icon = "heavyserv"
+			special_light_key = "heavyserv"
+
```
Comment (MCterra10) [inline code/modules/mob/living/silicon/robot/robot_modules.dm:646]: Surround these changes with
`// austation begin -- <reason for change>` on a line before the start of the modified code
and
`// austation end` on a line following the changes

File: code/modules/mob/living/silicon/robot/robot_modules.dm [hunk 2]
```diff
@@ -590,7 +687,7 @@
 
 /obj/item/robot_module/miner/be_transformed_to(obj/item/robot_module/old_module)
 	var/mob/living/silicon/robot/R = loc
-	var/borg_icon = input(R, "Select an icon!", "Robot Icon", null) as null|anything in sortList(list("Lavaland Miner", "Asteroid Miner", "Spider Miner"))
+	var/borg_icon = input(R, "Select an icon!", "Robot Icon", null) as null|anything in sortList(list("Lavaland Miner", "Asteroid Miner", "Spider Miner", "Marina", "Heavy", "Can", "Droid", "Sleek"))
```
Comment (MCterra10) [inline code/modules/mob/living/silicon/robot/robot_modules.dm:690]: Comment this

File: code/modules/mob/living/silicon/robot/robot_modules.dm [hunk 3]
```diff
@@ -601,6 +698,16 @@
...
+		if("Marina")
+			cyborg_base_icon = "marinamin"
+		if("Heavy")
+			cyborg_base_icon = "heavymin"
+		if("Can")
+			cyborg_base_icon = "canmin"
+		if("Droid")
+			cyborg_base_icon = "cminer"
+		if("Sleek")
+			cyborg_base_icon = "sleekmin"
```
Comment (MCterra10) [inline code/modules/mob/living/silicon/robot/robot_modules.dm:710]: comment these

Comment (MCterra10) [reply code/modules/mob/living/silicon/robot/robot_modules.dm:625]: lower case, space after the //, then after the austation start, a -- then the reason for the edit
austation end just needs a space after the // and lowercase

File: code/modules/mob/living/silicon/robot/robot_modules.dm (hunk 1 above)
Comment (MCterra10) [reply code/modules/mob/living/silicon/robot/robot_modules.dm:560]: lower case, space after the //, then after the austation start, a -- then the reason for the edit, also needs have have begin after the // austation
same as above, remove the reason, add a space after //, lower case, make it austation end

Comment (MCterra10) [reply code/modules/mob/living/silicon/robot/robot_modules.dm:555]: you misspelled austation, and add the `begin` after `// austation`

File: code/modules/mob/living/silicon/robot/robot_modules.dm (hunk 3 above)
Comment (MCterra10) [reply code/modules/mob/living/silicon/robot/robot_modules.dm:620]: misspelled austation, add the `begin` after the `// austation`

**Pull request merged**